remote_process_client.d

```diff
@@ -278,12 +278,12 @@ private:
         return bytes;
     }
 
-    void writeBytes (const ubyte [] bytes)
+    void writeBytes (const (ubyte) [] bytes)
```

Comment:
How would it write an `immutable ubyte []` array with this change?

In my understanding, the current `const ubyte []` universally accepts mutable and immutable for any portion of the type.